components/catalogs/back/tests/Integration/Infrastructure/Persistence/SearchAttributesQueryTest.php

```diff
@@ -39,6 +39,17 @@ public function testItGetsAllowedAttributesTypes(): void
                 'type' => 'pim_catalog_text',
                 'scopable' => false,
                 'localizable' => false,
+                'measurement_family' => null,
+                'default_measurement_unit' => null,
+            ],
+            [
+                'code' => 'weight',
+                'label' => '[weight]',
+                'type' => 'pim_catalog_metric',
+                'scopable' => false,
+                'localizable' => false,
+                'measurement_family' => null,
+                'default_measurement_unit' => null,
```

Comment:
```suggestion
                'measurement_family' => null,
                'default_measurement_unit' => null,
```

That's confusing. A `pim_catalog_metric` should always return values for those 2 attributes.